autotest/ogr/ogr_gml_citygml_read.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+# $Id$
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  GML Reading Driver for Japanese FGD GML v4 testing.
+# Author:   Hiroshi Miura <miurahr@linux.com>
+#
+###############################################################################
+# Copyright (c) 2017, Hiroshi Miura <miurahr@linux.com>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+###############################################################################
+
+
+
+import ogrtest
+from osgeo import gdal
+from osgeo import ogr
+from osgeo import osr
+import pytest
+
+
+###############################################################################
+# Test reading CityGML files
+###############################################################################
+
+_citygml_dir = 'data/gml_citygml/'
+
+###############################################################################
+# Test reading CityGML of Project PLATEAU
+
+def test_gml_read_compound_crs_lat_long():
+
+    # open CityGML file
+    gml = ogr.Open(_citygml_dir + 'compound_crs.gml')
+
+    # check number of layers
+    assert gml.GetLayerCount() == 1, 'Wrong layer count'
+
+    lyr = gml.GetLayer(0)
+    sr = osr.SpatialReference()
+    sr.ImportFromEPSG(6668)  # JGD2011
```

Comment:
I'm a bit confused why this isn't EPSG:6697, given the content of the GML file ?

Reply:
I was worried about this too.
Is this another bug?

Reply:
> Is this another bug?

not sure to remember if this is intended or not. I'd say probably not intended. there must be something somewhere that only keeps the horizontal part of the CRS. If you want to address that, that could be dealt as a follow-up fix after this PR is merged

Reply:
I'll look into this later.